stocks/Assets.xcassets/wallpaper_welcome.imageset/Contents.json

```diff
@@ -0,0 +1,89 @@
+{
```

Comment:
you shouldn't make images for light/dark mode. you only need one image (green rectangle).
the welcome text should be added in code (imagine if we needed to translate it)
for the background and text colors look up dynamic colors - they automatically adjust to changing between light/dark themes. (https://developer.apple.com/design/human-interface-guidelines/ios/visual-design/color/)